src/composables/use-ec-sorting/use-ec-sorting.ts

```diff
@@ -0,0 +1,50 @@
+import { ref, unref } from 'vue';
+
+import { SortDirection, SortDirectionCycle } from '../../enums';
+import type { SortingOptions } from './types';
+
+export default function useEcSorting({
+  initialSorts = [],
+  isMultiSort = false,
+  sortCycle: sortCycleInOptions,
+}: Partial<SortingOptions> = {}) {
+  const defaultSortCycle = unref(sortCycleInOptions) ?? SortDirectionCycle.LOWEST_FIRST;
+  // eslint-disable-next-line no-param-reassign
+  isMultiSort = unref(isMultiSort);
+
+  const sorts = ref(unref(initialSorts));
```

Comment:
and here, I think we don't need unref in all of them, or we need to change type of `SortingOptions`